docs/pages/inboxes/build-inbox.mdx

```diff
@@ -26,202 +26,189 @@ Learn to build a chat inbox with the help of [XMTP.chat](https://xmtp.chat/), an
-    signMessage: async (message) => {
-      // return value from a signing method here
+  const signer: Signer = { 
+    getIdentity: () => accountIdentity, 
+    signMessage: async (message) => { 
+      // return value from a signing method here 
     },
-    // these methods are required for smart contract wallets
-    // block number is optional
-    getBlockNumber: () => undefined,
-    // this example uses the Base chain
-    getChainId: () => BigInt(8453),
-  };
+    // These methods are required for smart contract wallets 
+    getBlockNumber: () => undefined, // Optional block number 
+    getChainId: () => BigInt(8453), // Example: Base chain ID 
+  }; 
   ```
 
-  ```tsx [React Native]
-  // Example SCW
-  export function convertSCWToSigner(scwAccount: SCWAccount): Signer {
-    return {
-      getAddress: async () => scwAccount.address,
-      signMessage: async (message: string) => {
-        const byteArray = await scwAccount.signMessage(message);
-        return ethers.utils.hexlify(byteArray); // Convert to hex string
+  ```tsx [React Native] 
+  // Example SCW Signer 
+  export function convertSCWToSigner(scwAccount: SCWAccount): Signer { 
+    return { 
+      getIdentity: async () => ({ 
+        kind: "ETHEREUM", // Specifies the identity type 
+        identifier: scwAccount.address, // Smart Contract Wallet address 
+      }),
+      signMessage: async (message: string) => { 
+        const byteArray = await scwAccount.signMessage(message); 
+        return ethers.utils.hexlify(byteArray); // Convert to hex string 
       },
-      walletType: () => "SCW",
-      getChainId: async () => 8453, // https://chainlist.org/
-      getBlockNumber: async () => undefined, // Optional: will be computed at run
-    };
-  }
+      getChainId: async () => 8453, // https://chainlist.org/ 
+      getBlockNumber: async () => undefined, // Optional: will be computed at runtime 
+    }; 
+  } 
   ```
 
-  ```kotlin [Kotlin]
-  class SCWallet : SigningKey {
-    override val address: String
-        get() = walletAddress
-
-    override val type: WalletType
-        get() = WalletType.SCW
-
-    override var chainId: Long? = 8453, // https://chainlist.org/
-
-    override var blockNumber: Long? = null, // Optional: will be computed at run
-
-    override suspend fun signSCW(message: String): ByteArray {
-        val digest = Signature.newBuilder().build().ethHash(message)
-        val replaySafeHash = smartWallet.replaySafeHash(digest).send()
-        val signature =
-            Sign.signMessage(replaySafeHash, contractDeployerCredentials.ecKeyPair, false)
-        val signatureBytes = signature.r + signature.s + signature.v
-        val tokens = listOf(
-            Uint(BigInteger.ZERO),
-            DynamicBytes(signatureBytes)
-        )
-        val encoded = FunctionEncoder.encodeConstructor(tokens)
-        val encodedBytes = Numeric.hexStringToByteArray(encoded)
-
-        return encodedBytes
-    }
+  ```kotlin [Kotlin] 
```

Comment:
As requested, add the missing `publicIdentity` based on the Swift sample - correct?

Reply:
Also pushed a commit for this.